internal/vrf/ips.go

```diff
@@ -0,0 +1,65 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) ips() *cobra.Command {
```

Comment:
While `ips` works, we've been exporting (arguably unnecessarily) the other subcommands. I don't see a reason to start deviating in this PR.